res/values-pt-rPT/strings.xml

```diff
@@ -596,7 +596,7 @@
 
     <string name="Enable_Custom_Skin">"Ativar Skin Personalizada"</string>
     <string name="Use_Custom_Skin">"skin\npersonalizada"</string>
-    <string name="custom_skin_description">"Este recurso permite que utilize a tua foto de perfil ou um ficheiro de imagem como skin. Isto só vai mudar a skin vista POR SI. As outras pessoas NÃO conseguirão ver a tua skin personalizada."</string>
+    <string name="custom_skin_description">"Este recurso permite utilizar a sua foto de perfil ou um ficheiro de imagem como skin, mas isso apenas mudará a skin visível para você. As outras pessoas NÃO conseguirão ver sua skin personalizada."</string>
```

Comment:
Partly wrong change.

1. **"a sua":** Portugal uses "a tua" to refer to the 2nd person (your). In Portugal, "a sua" means "his" or "her", that is 3rd person.
2. **"Isso":** it's not really right for Portugal, but "Isto" is.
3. "você":** Portugal doesn't use "você" in singular way, but "tu" or "ti".